### DISCUSSION TITLE
Issue with shipping service while running in terraform

### DISCUSSION THREAD

**Discussion by Annepuvarunkumar** [learndevopsonline/learndevopsonline#1622] [Q&A] [answered]
![Screenshot 2023-11-20 223551](https://github.com/learndevopsonline/learndevopsonline/assets/139056909/c3151021-36d4-48ab-8606-2b8a97767c04)

**Comment by sreedharm07** [comment]
@Annepuvarunkumar 
not able to login
can you turn on your instances and ping ips

**Reply by Annepuvarunkumar** [reply]
yes, i have turned on my instance

**Reply by sreedharm07** [reply]
to check issue please turn on instances and ping your ip here

**Reply by Annepuvarunkumar** [reply]
public ip:- 44.202.104.99

**Reply by sreedharm07** [reply]
Hi @Annepuvarunkumar 
before when you runned. mysql configuration was not runned thats why  its unable to load and the error was happened.
now i destroyed all things and runned not its runned without errors.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/98b916f4-266d-48e3-9d87-8e2a7b99faf7)

**Reply by Annepuvarunkumar** [reply]
yes i have checked it but its getting an error while placing an order

**Reply by sreedharm07** [reply, marked as answer]
hi @Annepuvarunkumar 
your rabbit mq password is roboshop123 you configured 
but
in payment you are accesing with different password please change it
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/ea43a2d2-f927-4ee9-ba82-176a6b290a02)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/3f0ef616-a53c-4cff-a269-548f70edd066)

**Reply by Annepuvarunkumar** [reply]
yes its working fine, thank you so much
